surveyo/README.md

```diff
@@ -32,15 +32,34 @@ Surveyo is a sample app that provides users with a survey tool that they can use
 
 ### Back-end
 
-- [Slash GraphQL](https://dgraph.io/slash-graphql)—a fully managed GraphQL backend service
+- [Slash GraphQL](https://dgraph.io/slash-graphql)—a fully managed GraphQL backend service.
+- [Using Auth0](https://dgraph.io/docs/learn/developer/todo-app-tutorial/todo-auth0-jwt/)—how to use auth0. 
 - [Auth0](https://auth0.com/)—Secure access for everyone.
 
+## Getting started
+
+### No-Auth Version
+
+1. Copy the entire contents of the `schema_noauth.graphql` file located in the root folder of the application.
+2. Open your `Graphql Client`, you can check some GraphQL clients [here](https://dgraph.io/docs/graphql/quick-start/#testing-your-graphql-api) or open a `terminal` and run the following command: `curl -X POST localhost:8080/admin/schema --data-binary <@SCHEMA_NOAUTH.GRAPHQL>`, which contains the code copied in step 1.
+3. To run the application locally, locate the `ApolloConfig.js` file in the application`src` folder and locate the `createApolloClient` function. Change the value of the constant `GRAPHQL ENDPOINT` to `http://localhost:8080/graphql`.
+4. In the root folder of the application, open a terminal and run the following commands: `npm install` followed by `npm start`.
+
+### Auth Version
+
+1. Copy the entire contents of the `schema_auth.graphql` file located in the root folder of the application.
+2. Open your `Graphql Client`, you can check some GraphQL clients [here](https://dgraph.io/docs/graphql/quick-start/#testing-your-graphql-api) or open a `terminal` and run the following command: `curl -X POST localhost:8080/admin/schema --data-binary <@SCHEMA.GRAPHQL>`, which contains the code copied in step 1.
+3. Create an authentication service account on: [Auth0](https://auth0.com/).
+4. After creating your Auth0 account, follow the instructions provided [here](https://dgraph.io/docs/learn/developer/todo-app-tutorial/todo-auth0-jwt/) to set up Auth0 authentication. 
+5. In the root folder of the application, open a terminal and run the following commands: `npm install` followed by `npm start`.
+
 ### Links
 
-- [Deploy Now](https://slash.dgraph.io/_/one-click?app=surveyo)
+<!-- - [Deploy Now](https://slash.dgraph.io/_/one-click?app=surveyo) -->
```

Comment:
Remove this comment. No need to left any references to Slash.

Reply:
Done.